src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup.Test/ScenarioTests/IaasVm/ContainerTests.ps1

```diff
@@ -12,31 +12,61 @@
 # limitations under the License.
 # ----------------------------------------------------------------------------------
 
+$resourceGroupName = "labRG1";
+$resourceName = "pstestrsvault";
+$vaultLocation = "westus";
```

Comment:
There are several examples of teams using this correctly.  Please remove the hard-coded locations.